ovos_utils/log.py

```diff
@@ -29,6 +29,8 @@
                 "ovos",
                 "phal",
                 "phal-admin",
+                "enclosure",
+                "admin",
```

Comment:
why this addition? 

those services are covered in the lines above as  "phal" and "phal-admin", this would be a duplication

Reply:
"enclosure" is what was used before OVOS moved to "phal" and what Neon still uses. The main motivation is about maintaining consistency and was previously about compat. with neon-cli/ovos-cli/mycroft-cli

Reply:
these are just the valid config keys, since it was never configurable with those "legacy" names there isn't a backwards compat concern here?

Reply:
we should minimize the alternative nomenclatures to reduce user confusion, this was introduced for the [ovos-logs util](https://github.com/OpenVoiceOS/ovos-utils/pull/203) which never supported the "legacy" names. so i don't think there is any expectation of backwards compat as nothing external should be checking that list anyway

ovos-cli-client has been deprecated and archived for a while too

Reply:
this list should be deleted with latest changes